federation/cluster/common.sh

```diff
@@ -92,8 +92,6 @@ function create-federation-api-objects {
     #We will use loadbalancer services where we can
     export FEDERATION_API_NODEPORT=32111
     export FEDERATION_NAMESPACE
-    export FEDERATION_NAME="${FEDERATION_NAME:-federation}"
```

Comment:
These 2 variables are already copied to federation-up.sh in previous commit. so we have duplicates in the path, so just removing them.